hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -502,6 +503,21 @@ private FlinkOptions() {
       .defaultValue(true) // default true for MOR write
       .withDescription("Schedule the compaction plan, enabled by default for MOR");
 
+  public static final ConfigOption<String> COMPACTION_SCHEDULE_SEQ = ConfigOptions
+      .key("compaction.schedule.seq")
+      .stringType()
+      .defaultValue(FlinkCompactionConfig.SEQ_FIFO) // default LIFO  to improve compact stability
+      .withDescription("Compaction plan execution sequence, two options are supported:\n"
+      + "1). FIFO: execute the oldest plan first;\n"
+      + "2). LIFO: execute the latest plan first, by default FIFO\n"
+      + "3). DYNAMIC: schedule the plan seq dynamic");
+
+  public static final ConfigOption<Integer> PENDING_PLAN_NUM = ConfigOptions
+      .key("pending.plan_num")
+      .intType()
```

Comment:
Can we avoid these options and make the strategy as default ? We can use a threshold here like 10 pending compaction plans to switch from `FIFO` to `LIFO`.

`compaction.fifo.max.pending.plans`

Default make the value 0 to mean we never switch that.